BUILD.bazel

```diff
@@ -77,3 +93,9 @@ go_image(
     pure = "on",
     visibility = ["//visibility:public"],
 )
+
+go_binary(
+    name = "cos-customizer",
+    embed = [":go_default_library"],
+    visibility = ["//visibility:public"],
+)
```

Comment:
I know that gazelle generates this rule, but we don't need it. So let's remove it.